Submitted By: Nick Fotopoulos (weasel at beyondnormal dot org)
Date: 2004-06-14
Initial Package Version: 2.1.10
Upstream Status: None
Origin: Self
Description: Adds prototype for new function that was added in this version


diff -Naur gal-2.1.10/gal/a11y/e-table/gal-a11y-e-table-item.h gal-2.1.10-fixed/gal/a11y/e-table/gal-a11y-e-table-item.h
--- gal-2.1.10/gal/a11y/e-table/gal-a11y-e-table-item.h	2003-11-03 01:40:50.000000000 -0500
+++ gal-2.1.10-fixed/gal/a11y/e-table/gal-a11y-e-table-item.h	2004-06-10 18:53:58.567602472 -0400
@@ -41,4 +41,6 @@
 					    ETableItem *item,
 					    int         index_in_parent);
 
+static gpointer *eti_reinit_data           (AtkTable *table,
+                                            ETableItem *item);
 #endif /* ! __GAL_A11Y_E_TABLE_ITEM_H__ */
